plugins/userBalance.ts

```diff
@@ -0,0 +1,7 @@
+export default ({ store }): void => {
+  const account = store.getters.getAuthAddress
+  if (account) {
+    // Subscribe to balance changes
+    store.dispatch('fetchBalance', account)
```

Comment:
// Subscribe to balance changes
and then function dispatch fetch?

😕 viki

Reply:
yes @vikiival looks like wrong comment message, it should be `Fetch and subscribe to balance changes`.